packages/core/src/components/loading-skeleton/_loading-skeleton.scss

```diff
@@ -0,0 +1,127 @@
+// Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+// of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+// and https://github.com/palantir/blueprint/blob/master/PATENTS
+
+@import "../../common/variables";
+
+.pt-loading-skeleton-bones {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+}
+
+@mixin bone-margin($margin) {
+  .pt-loading-skeleton-bone-#{$margin} {
+    margin-right: percentage($margin / 100);
```

Comment:
Conceptually this is great, but I feel weird about `margin-right`. Why not a `width` between 70% and 100%?